### DISCUSSION TITLE
When an audit can be expected?

### DISCUSSION THREAD

**Discussion by PovilasID** [netbirdio/netbird#565] [Issue Triage] [answered]
Hey,

I like the feature set but I can not use it for anything meaningful if there is not an audit behind it. New technology adoption is hard and if there is nobody (excluding founders) who would lose money if there was a problem that exposes companies to an imbalance of risk that they can not accept.

**Comment by mlsmaycon** [comment]
We are working with the beginning of Q1/2023 as the target for delivering the feature.

**Comment by mlsmaycon** [comment]
@PovilasID I just realized you are not talking about logs. Still we have planned for ISO  certification and external audit next year between the second and third quarter.

**Reply by PovilasID** [reply]
Could you link to the certification report?

**Reply by TechHutTV** [reply, marked as answer]
Here is a quick link to our [Trust Center](https://trust.netbird.io/). 😄
